data/json/furniture_and_terrain/furniture-fungal.json

```
@@ -61,5 +61,33 @@
     "required_str": -1,
     "flags": [ "FLAMMABLE_ASH", "FUNGUS" ],
     "bash": { "str_min": 26, "str_max": 50, "sound": "poof.", "sound_fail": "squelch." }
+  },
+  {
+    "type": "terrain",
+    "id": "t_fungus_colony",
+    "name": "mushroom colony",
+    "description": "A large pile of organic matter which has molded over with fungus.  The caps of many mushrooms spring from the top, all of which are fortunately terrestrial.  You could snip some of the mushrooms from it.",
```

```suggestion
    "description": "A large pile of organic matter which has molded over with fungus.  The caps of many mundane mushrooms spring from the top, you can even recognise some morels among them.  You could snip some of the mushrooms from it.",
```
Maybe something like that? The fact that there is a variety of mushroom like stuff that is not from earth in game is meta knowledge not every character will have encountered them before seeing this room